src/index.js

```diff
@@ -1,4 +1,4 @@
-const createWorkerMiddleware = (worker) => {
+const createWorkerMiddleware = (worker, meta = true) => {
```

Comment:
The variable name `meta` is confusing here, I think. Let's rename it to `workerName`. Also, I'll prefer to get rid of the `= true` default value and do a more explicit check below:

Reply:
✅